js/lib/ui.js

```diff
@@ -73,6 +75,11 @@ elgg.ui.toggles = function(event) {
 			$elem.toggle();
 		}
 	});
+
+	$(document).trigger('elgg.ui.toggle', [{
+		$toggler: $this,
+		$target: $target
+	}]);
```

Comment:
I could of course replace with with elgg.trigger_hook, and pass in the element/string instead of $-wrapped elements.